src/Utils/utils.ts

```diff
@@ -544,3 +544,29 @@ export const fahrenheitToCelsius = (fahrenheit: number) => {
 export const keysOf = <T extends object>(obj: T) => {
   return Object.keys(obj) as (keyof T)[];
 };
+
+//removes all symbols except _ and - (part of startCase)
+export const cleanString = (str: string): string => {
+  if (!str || str.length === 0) return "";
+  return str.replace(/[^a-zA-Z0-9_-]+/g, " ").trim();
+};
+
+// just capitalizes (as per startCase working) (part of startCase)
+export const capitalizeWords = (str: string): string => {
+  if (!str || str.length === 0) return "";
```

Comment:
Does this line add any value? It's equivalent to not having this line in the first place right?

Since we are following proper types, `str` would never be nullish. And `""` is falsey. 

<img width="671" alt="image" src="https://github.com/user-attachments/assets/f1a9be37-db76-48a8-825e-b7aa899d9748">


```suggestion
```